awx/main/tests/functional/models/test_notifications.py

```
@@ -131,6 +132,45 @@ def test_context_job_metadata_with_unicode(self):
+        notification_template.save()
+        job = Job.objects.create(name='test-job')
+        msg, body = job.build_notification_message(notification_template, 'running')
+        assert '/api/myprefix/v2/' in body
+
+    @pytest.mark.django_db
+    @override_settings(OPTIONAL_API_URLPATTERN_PREFIX='/myprefix/')
+    def test_build_notification_message_url_prefix_strips_slashes(self, notification_template):
+        notification_template.messages = {'started': {'message': '{{ job.url }}', 'body': '{{ job.url }}'}}
+        notification_template.save()
+        job = Job.objects.create(name='test-job')
+        msg, body = job.build_notification_message(notification_template, 'running')
```

Ruff (0.16.1)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)